packages/lib/modules/pool/actions/add-liquidity/form/AddLiquidityForm.tsx

```diff
@@ -103,6 +105,20 @@ function AddLiquidityMainForm() {
     setPriceImpact(priceImpactQuery.data)
   }, [priceImpactQuery.data])
 
+  useEffect(() => {
+    humanAmountsIn.forEach(amount => {
+      if (isUnbalancedJoinErrorMessage(priceImpactQuery.error) && amount.humanAmount) {
+        setValidationError(amount.tokenAddress, 'Amount causes pool imbalance')
+      }
+    })
+  }, [priceImpactQuery.error])
+
+  useEffect(() => {
+    humanAmountsIn.forEach(amount => {
+      setValidationError(amount.tokenAddress, '')
+    })
+  }, [humanAmountsIn])
```

Comment:
I'm worried about this overriding a `setValidationError` that is set in the token input. e.g. The user enters more than their balance, what's triggered first? This or the `setValidationError` within the token input? Could this cause a race condition?

Reply:
Looks like the '>balance' error isn't trigger any more now :(
Maybe remove the token input validation and add back the extra line to `disabledConditions`?
The warning msg is clear about what the user should do.

```
[isUnbalancedAddErrorMessage(priceImpactQuery.error), 'Unbalanced add'],
```

![image](https://github.com/user-attachments/assets/c24c4cc7-de89-4444-89ad-5f5c7d1757c4)

![image](https://github.com/user-attachments/assets/59e5842d-ef0a-47d8-9f79-100e21131b7b)

Reply:
Yes this is the best solution I think.